cmake/thirdparty/SetupCUDA.cmake

```diff
@@ -45,8 +45,22 @@
 ################################
 set (CMAKE_MODULE_PATH "${BLT_ROOT_DIR}/cmake/thirdparty;${CMAKE_MODULE_PATH}")
 
+
+if( ${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.9.0" )
```

Comment:
I think 3.8.0 might actually be fine.  Did you come up with this number from somewhere?

Reply:
I think  there is quite a big difference in "native" CUDA support between 3.8 and 3.9, @cyrush has recommended 3.9 in the past.

Reply:
I tried 3.9 and it worked. Tried 3.7 and it didn’t work. There wasn’t 3.8 on Ray. The cmake documentation for CMAKE_CUDA_HOST_COMPILER appears to start at 3.10 so I figured 3.9 would be the earliest that should be specified.